<!DOCTYPE html>
<html lang="en">
   <head>
      <meta charset="utf-8">
      <title>Chen Research Lab | Publications</title>
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <meta name="description" content="Chen Research Lab, Publications">
      <meta name="author" content="">
      <!-- Le styles -->
      <link href="css/bootstrap.min.css" rel="stylesheet">
      <link href="css/bootstrap-responsive.min.css" rel="stylesheet">
      <link href="css/theme.css" rel="stylesheet">
   </head>
   <body>
      <div class="container">
         <div class="headercontainer">
            <!-- div class="figure-large"><img src="images/lab-logo.png"></div -->
            <header class="jumbotron subhead" id="overview">
               <p class="lead"> University of Missouri </p>
               <h2>Computational RNA Biophysics</h2>
            </header>
            <!-- div class="figure-small"><a href="https://physics.missouri.edu/"><img src = "images/MU_StackedMU_4C.jpg"/></a></div -->
         </div>
         <div class="masthead">
            <div class="navbar">
               <div class="navbar-inner">
                 
                     <ul class="nav">
                        <li><a href="index.html">Home</a></li>
                        <li><a href="people.html">Team</a></li>
                        <li><a href="research.html">Research</a></li>
                        <li class="active"><a href="#">Publications</a></li>
                        <li><a href="software.html">Software</a></li>
                        <li><a href="contact.html">Contact</a></li>
                     </ul>
                  
               </div>
            </div>
         </div>
         <hr>
         <div class="row-fluid">
            <div class="span3 bs-docs-sidebar" id="navparent">
               <ul class="nav nav-list bs-docs-sidenav" data-spy="affix" data-offset-top="200" data-offset-bottom="260">
                  <li><a class="subhead" href="#2025"> 2025 </a></li>
                  <li><a class="subhead" href="#2024"> 2024 </a></li>
                  <li><a class="subhead" href="#2023"> 2023 </a></li>
                  <li><a class="subhead" href="#2022"> 2022 </a></li>
                  <li><a class="subhead" href="#2021"> 2021</a></li>
                  <li><a class="subhead" href="#2020"> 2000-2020</a></li>
               </ul>
            </div>
            <div class="span8 offset0">
                  <section id="2025">
                     <h3>2025</h3>

<p>Jin L, Liyanage R, Duan D, Chen SJ. (2026) Machine learning-inferred and energy landscape-guided analyses reveal kinetic determinants of CRISPR/Cas9 gene editing. PRX Life. In Press.  </p>
					  
<p>Romero RA, Chauvier A, Teh SS, Reed VA, Zhang S, Szyjka CE, Chen SJ, Strobel EJ, Walter NG. (2026) Co-transcriptional folding orchestrates sequential multi-effector sensing by a glycine tandem riboswitch.  Nature Communications. In Press.  </p>
					  
<p>Youhan Lee, Shujun He, Toshiyuki Oda, G John Rao, Yehyun Kim, Raehyun Kim, Hyunjin Kim, Cher Keng Heng, Danny Kowerko, Haowei Li, Hoa Nguyen, Arunodhayan Sampathkumar, Raul Enrique Gomez, Meng Chen, Atsushi Yoshizawa, Shun Kuraishi, Kenji Ogawa, Shuxian Zou, Alejo Paullier, Bingkang Zhao, Huey-Long Chen, Tsu-An Hsu, Tatsuya Hirano, Jeanine G. Gezelle, Daniel Haack, Yibao Hong, Shekhar Jadhav, Deepak Koirala, Rachael C. Kretsch, Anna Lewicka, Shanshan Li, Marco Marcia, Joseph Piccirilli, Boris Rudolfs, Yoshita Srivastava, Anna-Lena Steckelberg, Zhaoming Su, Navtej Toor, Liu Wang, Zi Yang, Kaiming Zhang, Jian Zou, David Baker, Shi-Jie Chen, Wah Chiu, Maggie Demkin, Andrew Favor, Alissa M. Hummer, Chaitanya K. Joshi, Andriy Kryshtafovych, Emine Kucukbenli, Zhichao Miao, John Moult, Christian Munley, Walter Reade, Theo Viel, Eric Westhof, Sicheng Zhang, Rhiju Das (2026). Template-based RNA structure prediction advanced through a blind code competition. bioRxiv 2025.12.30.696949; doi: https://doi.org/10.64898/2025.12.30.696949  </p>					 
					  
<p>Zhang SC, Chen SJ. (2026). Learning RNA Language Through Pre-trained Models: from Sequence to Structure. Biophysics Reviews. In Press.  </p>

<p>Kretsch, Rachael C and Posani, Elisa and Baulin, Eugene F and Bujnicki, Janusz M and Bussi, Giovanni and Cheatham, Thomas E and Chen, Shi-Jie and Elofsson, Arne and Farsani, Masoud A and Fisher, Olivia N and Gromiha, Michael M and Gupta, Ayush and Hamada, Michiaki and Harini, K. and Hu, Gang and Huang, David and Iwakiri, Junichi and Jain, Anika and Kagaya, Yuki and Kihara, Daisuke and Kmiecik, Sebastian and Krishnan, Sowmya R and Kurisaki, Ikuo and Languin-Cattoen, Olivier and Li, Jun and Li, Shanshan and Malekzadeh, Karim and Nakamura, Tsukasa and Ni, Wentao and Nithin, Chandran and Palo, Michael Z and Park, Joon H and Pilla, Smita P and Poblete, Simon and Pucci, Fabrizio and Punuru, Pranav and Saha, Anouka and Sato, Kengo and Srivastava, Ambuj and Terashi, Genki and Tugolukova, Emilia and Verburgt, Jacob and Wuyun, Qiqige and Zerze, Gul H and Zhang, Kaiming and Zhang, Sicheng and Zheng, Wei and Zhou, Yuanzhe and Chiu, Wah and Case, David A and Das, Rhiju. (2025) Blind Prediction of Complex Water and Ion Ensembles Around RNA in CASP16. ROTEINS: Structure, Function, and Bioinformatics. In Press. </p>

<p>Zhou YZ, Li WF, Chen SJ. (2025) RLDOCKScore: A Scoring Function for RNA-Ligand Docking and Small Molecule Virtual Screening. Journal of Chemical Theory and Computation. DOI: DOI: 10.1021/acs.jctc.5c00981   </p>

<p>Jin L, Chen SJ. (2025) Computational Modeling of Cotranscriptional RNA Folding. Computational and Structural Biotechnology Journal. In press. https://doi.org/10.1016/j.csbj.2025.06.005  </p>

<p>Liyanage R, Chen SJ. (2025) Computational advances in optimization of CRISPR gene editing system. Communications in Information and Systems. In press.  </p>

<p>Zhang SC, Li J, Zhou YZ, Chen SJ. (2025). Enhancing RNA 3D Structure Prediction in CASP16: Integrating Physics-Based Modeling with Machine Learning for Improved Predictions. PROTEINS: Structure, Function, and Bioinformatics. http://doi.org/10.1002/prot.26856 </p>

<p> Tian, K, Zhang S, Chen S, Chingarande, RG, Hou, CR, Ma, E, Ren, J, Ding, SH, Stertzer, M, Luang, BQ, Chenc, SJ, Chen, SY, Gu, LQ. (2025)
Advancing synthesis-free and enzyme-free rewritable DNA memory through frameshift encoding and nanopore duplex interruption decoding.
PNAS NEXUS, In Press.  </p>

 <p>Zhou Y, Chen SJ. (2025) Harnessing Computational Approaches for RNA-Targeted Drug Discovery. RNA Nanomed. 1(1):1–15. doi: 10.59566/isrnn.2024.0101001. PMCID: PMC11975998.  </p>

 <p>Pérez-López D, Burke M, Hakim C, Teixeira J, Han J, Yue Y, Ren Z, Sun J, Chen SJ, Herzog R, Yao G, Duan D. (2025) Circulatory CCL2 distinguishes Duchenne muscular dystrophy dogs. Disease Models & Mechanisms. 18(3):dmm052137. doi: 10.1242/dmm.052137. PMID: 40084478; PMCID: PMC12140648.  </p>


                  </section>
                  <hr>
                  <section id="2024">
                     <h3>2024</h3>
                      <p>Jin L, Zhou YZ, Zhang SC, Chen SJ. (2024) mRNA Vaccine Sequence and Structure Design and Optimization: Advances and Challenges. Journal of Biological Chemistry. 301(1):108015. doi: 10.1016/j.jbc.2024.108015. PMID: 39608721.  </p>

                     	   <p>Bu F, Adam Y, Adamiak RW, Antczak M, de Aquino BRH, Badepally NG, Batey RT, Baulin EF, Boinski P, Boniecki MJ, Bujnicki JM, Carpenter KA, Chacon J, Chen SJ, Chiu W, Cordero P, Das NK, Das R, Dawson WK, DiMaio F, Ding F, Dock-Bregeon AC, Dokholyan NV, Dror RO, Dunin-Horkawicz S, Eismann S, Ennifar E, Esmaeeli R, Farsani MA, Ferré-D'Amaré AR, Geniesse C, Ghanim GE, Guzman HV, Hood IV, Huang L, Jain DS, Jaryani F, Jin L, Joshi A, Karelina M, Kieft JS, Kladwang W, Kmiecik S, Koirala D, Kollmann M, Kretsch RC, Kurciński M, Li J, Li S, Magnus M, Masquida B, Moafinejad SN, Mondal A, Mukherjee S, Nguyen THD, Nikolaev G, Nithin C, Nye G, Pandaranadar Jeyeram IPN, Perez A, Pham P, Piccirilli JA, Pilla SP, Pluta R, Poblete S, Ponce-Salvatierra A, Popenda M, Popenda L, Pucci F, Rangan R, Ray A, Ren A, Sarzynska J, Sha CM, Stefaniak F, Su Z, Suddala KC, Szachniuk M, Townshend R, Trachman RJ 3rd, Wang J, Wang W, Watkins A, Wirecki TK, Xiao Y, Xiong P, Xiong Y, Yang J, Yesselman JD, Zhang J, Zhang Y, Zhang Z, Zhou Y, Zok T, Zhang D, Zhang S, Zyla A, Westhof E, Miao Z. RNA-Puzzles Round V: blind predictions of 23 RNA structures. Nat Methods. 2024 Dec 2. doi: 10.1038/s41592-024-02543-9. Epub ahead of print. PMID: 39623050.  </p>

                           <p>Zhou YZ, Jiang YW, Chen SJ. (2024) SPRank - A knowledge-based scoring function for RNA-ligand pose prediction and virtual screening. Journal of Chemical Theory and Computation.  doi: 10.1021/acs.jctc.4c00681. PMID: 39150889. </p>

                           <p>Zhou YZ, Fichman Y, Zhang, SC, Mittler R, Chen, SJ. (2024) Modeling the Reactive Oxygen Species (ROS) wave in Chlamydomonas reinhardtii colonies. Free Radic Biol Med. 2024 Jun 6;222:165-172. doi: 10.1016/j.freeradbiomed.2024.06.003. </p>

                           <p>Avila YI, Rebolledo LP, Skelly E, Saito Rde F, Wei H, Lilley D, Stanley RE, Hou YM, Yang H, Sztuba-Solinska J, Chen SJ, Dokholyan NV, Tan C, Li K, He X, Zhang X, Miles W, Franco E, Binzel DW, Guo P, Afonin KA. (2024) Cracking the Code: Enhancing Molecular Tools for Progress in Nanobiotechnology. ACS Appl Bio Mater. 2024 Jun 17;7(6):3587-3604. doi: 10.1021/acsabm.4c004. </p>

                           <p>	Li, J., Zhou, YZ., Chen, SJ. (2024). Embracing exascale computing in nucleic acid simulations. Current Opinion in Structural Biology. 87:102847. doi: 10.1016/j.sbi.2024.102847. Epub 2024 May 29. PMID: 38815519; PMCID: PMC11283969. </p> 
                           </a>
                           <p> Jin, L., Zhang, SC., Song, ZW., Heng, X., Chen, SJ. (2024). Kinetic pathway of HIV-1 TAR cotranscriptional folding. Nucleic Acids Research. DOI: 10.1093/nar/gkae362 </p> 
			   <p> Li, J, Walter, N.G., and Chen, S. (2024) smFRET-assisted RNA structure prediction. Communications in Information and Systems (CIS). Vol. 24 No. 3: 163-179.  </p>
			   <p> Zhou, YZ and Chen S. (2024) Advances in machine-learning approaches to RNA-targeted drug design. Artificial Intelligence Chemistry. 2(1):100053. doi: 10.1016/j.aichem.2024.100053.</p>
			   <p> Fichman Y, Rowland L, Nguyen T, Chen S, Mittler R. (2024) Propagation of a rapid cell-to-cell H2O2 signal over long distances in a monolayer of cardiomyocyte cells. Redox Biology. 70:103069. doi: 10.1016/j.redox.2024.103069. </p>
			   <p> Zhang, SC.<sup>+</sup>, Li, J.<sup>+</sup>, Chen, SJ. (2024). Machine Learning in RNA Structure Prediction: Advances and Challenges. Biophysical Journal. doi: 10.1016/j.bpj.2024.01.026. </p>
                  </section>
                  <hr>
                  <section id="2023">
                     <h3>2023</h3>
                           <p> Li, J.<sup>+</sup>, Zhang, SC.<sup>+</sup>, Chen, SJ. (2023). Advancing RNA 3D Structure Prediction: Exploring Hierarchical and Hybrid Approaches in CASP15. PROTEINS: Structure, Function, and Bioinformatics. 91(12):1779-1789. Open Access: http://doi.org/10.1002/prot.26583 </p>
			   <p> Li, J., & Chen, S. (2023). RNAJP: enhanced RNA 3D structure predictions with noncanonical interactions and global topology sampling. Nucleic Acids Research, 51(7):3341-3356. doi: 10.1093/nar/gkad122. </p>
			   <p> Chen, S. J., Hassan, M., Jernigan, R. L., Jia, K., Kihara, D., Kloczkowski, A., Kotelnikov, S., Kozakov, D., Liang, J., Liwo, A., Matysiak, S., Meller, J., Micheletti, C., Mitchell, J. C., Mondal, S., Nussinov, R., Okazaki, K. I., Padhorny, D., Skolnick, J., Sosnick, T. S., Rose, G. D. (2023). Opinion: Protein folds vs. protein folding: Differing questions, different challenges. Proc Natl Acad Sci U S A., 120(1), e2214423119. https://doi.org/10.1073/pnas.2214423119 </p>

                  </section>
                  <hr>
                  <section id="2022">
                     <h3>2022</h3>
                     <p> Lewis Rolband, Damian Beasock, Yang Wang, Yao-Gen Shu, Jonathan D. Dinman, Tamar Schlick, Yaoqi Zhou, Jeffrey S. Kieft, Shi-Jie Chen, Giovanni Bussi, Abdelghani Oukhaled, Xingfa Gao, Petr Šulc, Daniel Binzel, Abhjeet S. Bhullar, Chenxi Liang, Peixuan Guo, Kirill A. Afonin. Biomotors, viral assembly, and RNA nanobiotechnology: Current achievements and future directions,
Computational and Structural Biotechnology Journal, Volume 20, 2022, Pages 6120-6137, https://doi.org/10.1016/j.csbj.2022.11.007. </p>
		     <p> Zhou, Y., & Chen, S. (2022). Graph deep learning locates magnesium ions in RNA. QRB Discovery, 3, E20. doi:10.1017/qrd.2022.17 </p>
		     <p> Samuelian Jr. JS, Gremminger TJ, Song ZW, Poudyal RR, Li J, Zhou YZ, Staller SA, Carballo JA, Roychowdhury-Saha M, Chen SJ, Burke DH, Heng X, Baum DA. An RNA aptamer that shifts the reduction potential of metabolic cofactors. Nat Chem Biol 18, 1263–1269 (2022). https://doi.org/10.1038/s41589-022-01121-4 </p>
		     <p> Gumna J, Antczak M, Adamiak RW, Bujnicki JM, Chen SJ, Ding F, Ghosh P, Li J, Mukherjee S, Nithin C, Pachulska-Wieczorek K, Ponce-Salvatierra A, Popenda M, Sarzynska J, Wirecki T, Zhang D, Zhang S, Zok T, Westhof E, Miao Z, Szachniuk M, Rybarczyk A. Computational Pipeline for Reference-Free Comparative Analysis of RNA 3D Structures Applied to SARS-CoV-2 UTR Models. Int J Mol Sci. 2022 Aug 25;23(17):9630. doi: 10.3390/ijms23179630. </p>
		     <p> Li J, Zhang S, Zhang D, Chen SJ. Vfold-Pipeline: a web server for RNA 3D structure prediction from sequences. Bioinformatics. 2022; 38(16): 4042-3. doi: 10.1093/bioinformatics/btac426. </p>
 		     <p> Wasala, N, Million, E.  Watkins, T.  Wasala, L.  Han, J.  Yue, Y.  Lu, B.  Chen, SJ.  Hakim, C.  Duan, D. The gRNA vector level determines the outcome of systemic AAV CRISPR therapy for Duchenne muscular dystrophy. Hum Gene Ther. 2022;33(9-10):518-528. doi:10.1089/hum.2021.130 </p> 
		     <p> Xu XJ<sup>+</sup>, Jin L<sup>+</sup>, Xie LX, and Chen SJ. Landscape-zooming towards the prediction of RNA cotranscriptional folding. J Chem Theory Comput. 2022;18(3)2002-2015. doi:10.1021/acs.jctc.1c01233 </p>
		     <p>  Zhang SC<sup>+</sup>, Cheng Y<sup>+</sup>, Guo PX, and Chen, SJ. VfoldMCPX: Predicting multistrand RNA complexes.  RNA. 2022;28(4):596-608. doi:10.1261/rna.079020.121 </p>
		     <p> Jiang, YW. and Chen, SJ. RLDOCK method for predicting RNA-small molecule binding modes. Methods. 2022. 197:97-105. doi: 10.1016/j.ymeth.2021.01.009 </p>
		     <p> Cheng Y, Zhang D, Hurst T, Zou XQ, Giangrande P, Chen SJ. RNA structural modeling for therapeutic applications. RNA Nanotechnology and Therapeutics, 2nd Edition (2022), ISBN 9781138312869, Taylor & Francis </p>
                  </section>
               <hr>
               <section id="2021">
                     <h3>2021</h3>
                     <p> Hakim CH, Kumar S, Perez-López D, Wasala N, Zhang D, Yue Y, Teixeira J, Pan X, Zhang K, Million E, Nelson C,Metzger S, Han J, Louderman J, Schmidt F, Feng F, Grimm D, Smith B, Yao G, Yang N, Gersbach C, Chen S-J, Herzog R, and Duan D. Cas9-specific immune responses compromise local and systemic AAV CRISPR therapy in multiple dystrophic canine models. Nat Commun 12, 6769 (2021). https://doi.org/10.1038/s41467-021-26830-7 (Commentaries: https://pubmed.ncbi.nlm.nih.gov/34895501/; https://www.liebertpub.com/doi/10.1089/hum.2021.29193.amk) </p>
		     <p> Zhang D, Chen SJ, and Zhou RH. Modeling noncanonical RNA base pairs by coarse-grained IsRNA2 model. J. Phys. Chem. B 2021, 125, 43, 11907-11915 </p>
		     <p>  Cheng Y<sup>+</sup>, Zhang SC<sup>+</sup>, Xu XJ, and Chen SJ. Vfold2D-MC: A physics-based hybrid model for predicting RNA secondary structure folding. J. Phys. Chem. B 2021, 125, 36, 10108-10118 </p>
		     <p> Zhou, YZ.<sup>+</sup>, Jiang, YW.<sup>+</sup>, Chen, SJ. RNA-ligand molecular docking: advances and challenges. Wiley Interdisciplinary Reviews (WIREs) Computational Molecular Science 2021. doi: 10.1002/wcms.1571 </p> 
		     <p> Zhou YZ, LI J, Hurst TH, and Chen SJ. SHAPER: A Web Server for Fast and Accurate SHAPE Reactivity Prediction. Front. Mol. Biosci. 2021. doi: 10.3389/fmolb.2021.721955 </p> 
		     <p> Li, J., Chen, SJ. RNA 3D structure prediction using coarse-grained models. Front. Mol. Biosci. 2021. 8:720937. doi: 10.3389/fmolb.2021.720937. </p> 
		     <p> Xu X., Chen SJ. (2021) Predicting RNA Scaffolds with a Hybrid Method of Vfold3D and VfoldLA. In: Ponchon L. (eds) RNA Scaffolds. Methods in Molecular Biology, 2323: 1-11. Humana, New York, NY. https://doi.org/10.1007/978-1-0716-1499-0_1
		     <p> Song, Zhenwei; Gremminger, Thomas; Singh, Gatikrushna; Cheng, Yi; Li, Jun; Qiu, Liming; Ji, Juan; Lange, Margaret; Zuo, Xiaobing; Chen, Shi-Jie; Zou, Xiaoqin ; Boris-Lawrie, Kathleen; Heng, Xiao. (2021) The three-way junction structure of the HIV-1 PBS-segment binds host enzyme important for viral infectivity.Nucleic Acids Research. 49(10):5925-5942. doi: 10.1093/nar/gkab342. </p>
		     <p> Chen, SJ. Graph, Pseudoknot, and SARS-Cov-2 Genomic RNA - a Biophysical Synthesis. Biophysical Journal. 2021, 120(6): 980-982. doi: 10.1016/j.bpj.2021.01.030 </p>
                     <p> Zhang, D.<sup>+</sup>, Li, J.<sup>+</sup>, Chen, SJ. (2021)  IsRNA1: de novo prediction and blind screening of RNA 3D structures. Journal of Chemical Theory and Computation. 17(3): 1842-1857. https://doi.org/10.1021/acs.jctc.0c01148. </p>
                     <p> Hurst,TH. and Chen, SJ. Deciphering nucleotide modification-induced structure and stability changes. RNA Biology, 2021, 18 (11): 3015-3026. DOI: 10.1080/15476286.2021.1882179 </p>
		     <p> Hurst,TH. and Chen, SJ. Sieving RNA 3D structures with SHAPE and evaluating mechanisms driving sequence-dependent reactivity bias. The Journal of Physical Chemistry B, 2021, 125, 4, 1156-1166. DOI: 10.1021/acs.jpcb.0c11365 </p>
		     <p> Hurst,TH., Zhang, D., Zhou, YZ., Chen, SJ. A Bayes-inspired Theory for Optimally Building an Efficient Coarse-grained Folding Force Field. Communications in Information and Systems, 2021 Volume 21 Number 1 Pages: 65-83 DOI: https://dx.doi.org/10.4310/CIS.2021.v21.n1.a4 </p>
               </section>
               <hr>
               <section id="2020">
                     <h3>2000-2020</h3>
                     <p> Sun, LZ.<sup>+</sup>, Jiang, YW.<sup>+</sup>, Zhou, YZ., Chen, SJ. RLDOCK: A new method for predicting RNA-ligand interactions. Journal of Chemical Theory and Computation, 2020 Nov 10;16(11):7173-7183. doi: 10.1021/acs.jctc.0c00798. </p>
		     <p> Zhao, CH<sup>+</sup>, Zhang, D<sup>+</sup>, Jiang, YW, Chen, SJ. Modeling loop composition and ion concentration effects in RNA hairpin folding stability. Biophys J. Volume 119, Issue 7, 6 October 2020, Pages 1439-1455. [<a href="https://www.sciencedirect.com/science/article/pii/S0006349520306767"><font color="#330099">See also "New and Notable": Petr Sulc. The Multiscale Future of RNA Modeling</font></a>] </p>
		     <p> Ptacek, Jakub; Zhang, Dong; Qiu, Liming; Kruspe, Sven; Motlova, Lucia; Kolenko, Petr; Novakova, Zora; shubham, shambhavi; Havlinova, Barbora; Baranova, Petra; Chen, Shi-Jie; Zou, Xiaoqin ; Giangrande, Paloma; Barinka, Cyril.Structural Basis of Prostate-specific Membrane Antigen Recognition by the A9g RNA Aptamer.Nucleic Acids Research,  Volume 48, Issue 19, 4 November 2020, Pages 11130–11145, https://doi.org/10.1093/nar/gkaa494 </p>
		     <p> Xu, XJ. and Chen, SJ. Topological constraints of RNA pseudoknotted and loop-kissing motifs: applications to three-dimensional structure prediction. Nucleic Acids Research, 2020, 48(12):6503-6512. doi: 10.1093/nar/gkaa463. </p>
		     <p> Miao Z, Adamiak RW, .... Chen SJ, ... Westhof E. (2020) RNA-Puzzles Round IV: 3D structure predictions of four ribozymes and two aptamers. RNA. 26(8):982-995. doi: 10.1261/rna.075341.120. </p>
		     <p> Wasala NB, Chen SJ, Duan D. Duchenne muscular dystrophy animal models for high-throughput drug discovery and precision medicine. Expert Opinion on Drug Discovery, 2020, 15(4):443-456. doi: 10.1080/17460441.2020.1718100.  </p>
		     <p> Nguyen PDM, Zheng J, Gremminger TJ, Qiu LM, Zhang D, Tuske S, Lange MJ, Griffin PR, Arnold E, Chen SJ, Zou XQ, Heng X, Burke DH. Binding interface and impact on protease cleavage for an RNA aptamer to HIV-1 reverse transcriptase. Nucleic Acids Research, 2020, 1-17. doi: 10.1093/nar/gkz1224. </p>
		     <p> Hurst,TH., Zhou, YZ., Chen, SJ. Analytical Modeling and Deep Learning Approaches to Estimating RNA SHAPE Reactivity from 3D Structure. Communications in Information and Systems. 2019; 19(3): 299-320. DOI: 10.4310/CIS.2019.v19.n3.a4. </p>
		     <p> Wang, FF., Sun, LZ., Cai, PG., Chen, SJ., Xu, XJ. Kinetic mechanism of RNA helix-terminal base pairing - a kinetic minima network analysis.Biophys J. 2019; 117(9): 1674-1683. doi: 10.1016/j.bpj.2019.09.017. </p>
		     <p> Nance ME, Shi R, Hakim CH, Wasala NB, Yue Y, Pan X, Zhang T, Robinson CA, Duan SX, Yao G, Yang NN, Chen SJ, Wagner KR, Gersbach CA, Duan D. AAV9 Edits Muscle Stem Cells in Normal and Dystrophic Adult Mice. Molecular Therapy. 2019. 27(9): 1568-1585. PMID 31327755 DOI: 10.1016/j.ymthe.2019.06.012  </p>
		     <p> Sun, LZ., Zhou, YZ., Chen, SJ. Predicting Monovalent Ion Correlation Effects in Nucleic Acids. ACS Omega. 2019; 4(8):13435-13446. doi: 10.1021/acsomega.9b01689. </p>
		     <p> Chen, SJ. Minimizing Off-target Effects in CRISPR-Cas9 Genome Editing. Cell Biology and Toxicology, 2019; 35(5), 399-401. doi: 10.1007/s10565-019-09486-4.</p>
		     <p> Xu, XJ., Zhao, CH., Chen, SJ. VfoldLA: a web server for loop assembly-based prediction of putative 3D RNA structures. J Struct Biol. 2019; 207(3): 235-240. doi: 10.1016/j.jsb.2019.06.002. </p>
		     <p> Zhang, D., Hurst, T., Duan, D., Chen, SJ. Unified energetics analysis unravels SpCas9 cleavage activity for optimal gRNA design. Proc Natl Acad Sci U S A. 2019, 116(18): 8693-8698.</p>
		     <p> Nalinda B. Wasala, Chady H. Hakim, Shi-Jie Chen, N. Nora Yang, Dongsheng Duan.Questions answered and unanswered by the first CRISPR editing study in the canine model of Duchenne muscular dystrophy. Human Gene Therapy 2019 Feb 26. doi: 10.1089/hum.2018.243. </p>
		     <p> Sun L-Z, Chen S-J, Predicting RNA-Metal Ion Binding with Ion Dehydration Effects, Biophysical Journal, 2019. 116(2):184-195. doi: 10.1016/j.bpj.2018.12.006. </p>
		     <p> Chady H. Hakim, Nalinda B. Wasala, Nelson E. Christopher, Lakmini Wasala, Yongping Yue, Jacqueline Louderman, Thais B Lessa, Aihua Dai, Keqing Zhang, Gregory J Jenkins, Michael E. Nance, Xiufang Pan, Kasun Kodippili, Na Nora Yang, Shi-Jie Chen, Charles Gersbach, Dongshen Duan. (2018) AAV CRISPR editing rescues cardiac and muscle function for 18 months in dystrophic mice.  
JCI Insight. 2018 Dec 6;3(23). pii: 124297. doi: 10.1172/jci.insight.124297.</p>
		    <p> Sun, TT<sup>+</sup>, Zhao, CH<sup>+</sup>, Chen, SJ. Predicting Cotranscriptional Folding Kinetics for Riboswitch.J. Phys. Chem. B. 2018, 122: 7484-7496. doi: 10.1021/acs.jpcb.8b04249.</p>
		    <p> Hurst, T, Xu, XJ, Zhao, PN, Chen, SJ. Quantitative Understanding of SHAPE Mechanism from RNA Structure and Dynamics Analysis. J. Phys. Chem. B. 2018, 122: 4771-4783. DOI: 10.1021/acs.jpcb.8b00575 </p>
		    <p> Zhang, D and Chen, SJ. IsRNA: An iterative simulated reference state approach to modeling correlated interactions in RNA folding. J. Chem. Theory Comput. 2018, 14: 2230–2239.
DOI: 10.1021/acs.jctc.7b01228 </p>
		    <p> Yu, T, Chen, SJ. Hexahydrated Mg<sup>2+</sup> Ion Binding and Outer-Shell Dehydration on RNA Surface. Biophys J. 2018, 114: 1274-1284. doi: 10.1016/j.bpj.2018.01.040. </p>
		    <p> Xu, X, Chen, SJ. Hierarchical Assembly of RNA Three-Dimensional Structures Based on Loop Templates. J. Phys. Chem. B 2018, 122: 5327-5335. DOI: 10.1021/acs.jpcb.7b10102 </p>
		    <p> Sun, LZ, Heng, X.<sup>*</sup>, Chen, SJ.<sup>*</sup> (2017) Theory Meets Experiment: Metal Ion Effects in HCV Genomic RNA Kissing Complex Formation. Front. Mol. Biosci. 4: 92. doi: 10.3389/fmolb.2017.00092 [<a href="https://doi.org/10.3389/fmolb.2017.00092"><font color="#330099">Article</font></a>] </p>
		    <p> Xinyue Zhang<sup>+</sup>, Dong Zhang<sup>+</sup>, Chenhan Zhao, Kai Tian, Ruicheng Shi, Xiao Du, Andrew Burcke, Jing Wang, Shi-Jie Chen<sup>*</sup>, and Li-Qun Gu<sup>*</sup> (2017) Nanopore electric snapshots of an RNA tertiary folding pathway. Nature Communications. 8: 1458. doi:10.1038/s41467-017-01588-z. </p>
		    <p> Kranawetter, C., Brady, S., Sun, LZ., Schroeder, M., Chen, SJ.<sup>*</sup>, Heng, X.<sup>*</sup> (2017) NMR study of RNA structures in the 3´-end of the Hepatitis C Virus genome. Biochemistry. 56: 4972-4984. </p>
		    <p> Sun, LZ, Kranawetter, C., Heng, X.<sup>*</sup>, Chen, SJ.<sup>*</sup> (2017) Predicting Ion Effects in RNA Conformational Equilibrium.The Journal of Physical Chemistry-B. 121: 8026-8036.</p>
		    <p> Sun, LZ, Zhang, JX, Chen, SJ. (2017) MCTBI: a web server for predicting metal ion effects in RNA structures.
RNA. 23: 1155-1165. doi: 10.1261/rna.060947.117. [<a href="http://f1000.com/prime/727617467?subscriptioncode=7f5b40b7-48fe-4819-baa3-2d1ae8e1c0e5&utm_medium=email&utm_source=prime_ypp"><font color="#330099">Faculty of 1000 Recommended</font></a>] </p>
		    <p> Xu, X, Duan, DS, Chen, SJ. (2017) CRISPR-Cas9 cleavage efficiency correlates strongly with target-sgRNA folding stability: from physical mechanism to off-target assessment.Sci Rep. 7(1):143. doi: 10.1038/s41598-017-00180-1. </p>
		    <p> Sun, LZ<sup>+</sup>, Zhang, D<sup>+</sup>, Chen, SJ. (2017) Theory and Modeling of RNA Structure and Interactions with Metal Ions and Small Molecules. Annu Rev Biophys. 46:227-246. doi: 10.1146/annurev-biophys-070816-033920. </p>
		    <p> Sun, LZ and Chen, SJ. (2017) A new method to predict ion effects in RNA folding. Methods Mol Biol. 1632: 1-17. doi: 10.1007/978-1-4939-7138-1_1. </p>
		    <p> Zhao, CH, Xu, XJ, Chen SJ. (2017) Predicting RNA structure with Vfold. Methods in Molecular Biology, 1654: 3-15. </p>
		    <p> Miao Z, Adamiak RW, Antczak M, Batey RT, Becka AJ, Biesiada M, Boniecki MJ, Bujnicki J, Chen SJ, Cheng CY, Chou FC, Ferre-D'Amare AR, Das R, Dawson WK, Feng D, Dokholyan NV, Dunin-Horkawicz S, Geniesse C, Kappel K, Kladwang W, Krokhotin A, Łach GE, Major F, Mann TH, Magnus M, Pachulska-Wieczorek K, Patel DJ, Piccirilli JA, Popenda M, Purzycka KJ, Ren A, Rice GM, Santalucia J Jr, Sarzynska J, Szachniuk M, Tandon A, Trausch JJ, Tian S5, Wang J, Weeks KM, Williams B 2nd, Xiao Y, Xu X, Zhang D, Zok T, Westhof E. (2017) RNA-Puzzles Round III: 3D RNA structure prediction of five riboswitches and one ribozyme. RNA. 23(5):655-672. doi: 10.1261/rna.060368.116. </p>
		    <p> Xu, X and Chen, SJ. (2016) VfoldCPX Server: Predicting RNA-RNA Complex Structure and Stability. PLoS One. 11(9):e0163454. doi: 10.1371/journal.pone.0163454. </p>
		    <p> Yu T, Zhu Y, He Z, Chen SJ. (2016) Predicting Molecular Crowding Effects in Ion-RNA Interactions. J Phys Chem B. 120(34):8837-44. doi: 10.1021/acs.jpcb.6b05625. PMID: 27490487. </p>
		    <p> Sun, LZ and Chen, SJ. (2016) Monte Carlo Tightly Bound Ion Model: Predicting Ion-Binding Properties of RNA with Ion Correlations and Fluctuations.J Chem Theory Comput. 12(7):3370-81. doi: 10.1021/acs.jctc.6b00028. </p>
		    <p> Xu, XJ., Dickey, DD., Chen, SJ., Giangrande, P. (2016) Structural Computational Modeling of RNA Aptamers.  Methods. 103:175-9. doi: 10.1016/j.ymeth.2016.03.004.
<!--In: Stuart Le Grice (ed) 
Advances in RNA Structure Determination. Methods. In Press. -->
</p>
		    <p> Urak KT, Shore S, Rockey WM, Chen SJ, McCaffrey AP, Giangrande PH. (2016) In vitro RNA SELEX for the generation of chemically-optimized therapeutic RNA drugs. Methods. 103: 167-174. doi: 10.1016/j.ymeth.2016.03.003 </p>
		    <p> Xu, XJ., Chen, SJ. (2016) A method to predict the structure and stability of RNA/RNA complexes. Methods Mol Biol. 1490:63-72. doi: 10.1007/978-1-4939-6433-8_5.
<!-- In: Douglas H.  Turner and David H.  Mathews (ed) 
RNA Structure Determination: Methods and Protocols. 
Methods in Molecular Biology, In Press. -->
</p>
		    <p> Xu, XJ., Yu, T., Chen, SJ. (2016) Understanding the kinetic mechanism of RNA single base pair formation. Proc Natl Acad Sci U S A. 113(1):116-21. doi: 10.1073/pnas.1517511113.[<a href="paps/2015PNAS_base_kin.pdf"><font color="#330099">PDF</font></a>] [<a href="http://f1000.com/prime/726037634"><font color="#330099">Faculty of 1000 Recommended</font></a>] </p>
		    <p> Zhang, XY., Xu, XJ., Yang, ZY., Burcke, AJ., Gates, KS., Chen, SJ.<sup>*</sup>, Gu, LQ.<sup>*</sup> (2015) Mimicking Ribosomal Unfolding of RNA Pseudoknot in a Protein Channel. 
J. Am. Chem. Soc. 2015, 137, 15742−15752. DOI: 10.1021/jacs.5b07910. [<a href="paps/2015JACS_pk_pore.pdf"><font color="#330099">PDF</font></a>]  <sup>*</sup>Corresponding authors.</p>
		    <p> Xu, XJ., Chen, SJ. (2015) Physics-based RNA structure prediction. Biophysics Reports. DOI: 10.1007/s41048-015-0001-4. [<a href="paps/2015Biophys_Report.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Zhichao Miao, Ryszard W. Adamiak, Marc-Frederick Blanchet, Michal Boniecki, Janusz M. Bujnicki, Shi-Jie Chen, Clarence Cheng, Grzegorz Chojnowski, Fang-Chieh Chou, Pablo Cordero, Jose Almeida Cruz, Adrian R. Ferre-D'Amare, Rhiju Das, Feng Ding, Nikolay V. Dokholyan, Stanislaw Dunin-Horkawicz, Wipapat Kladwang, Andrey Krokhotin, Grzegorz Lach, Marcin Magnus, François Major, Thomas H. Mann, Benoit Masquida, Dorota Matelska, Melanie Meyer, Alla Peselis, Mariusz Popenda, Katarzyna J. Purzycka, Alexander Serganov, Juliusz Stasiewicz, Marta Szachniuk, Arpit Tandon, Siqi Tian, Jian Wang, Yi Xiao, Xiaojun Xu, Jinwei Zhang, Peinan Zhao, Tomasz Zok, and Eric Westhof (2015) RNA-Puzzles Round II: assessment of RNA structure prediction programs applied to three large RNA structures.RNA, 21:1-19. [<a href="paps/2015RNA_puzzle2_RNA.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Zhu, YH, He, ZJ., Chen, SJ. (2015) TBI server: a web server for predicting ion effects in RNA folding. PLoS ONE doi: 10.1371/journal.pone.0119705. [<a href="paps/2014PLoS1VfoldServer.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Shi-Jie Chen and Donald Burke (ed) (2015) Computational Methods for Understanding Riboswitches, Volume 553, Methods in Enzymology, Academic Press. ISBN-13: 978-0128014295, ISBN-10: 0128014296. [<a href="http://www.elsevier.com/books/computational-methods-for-understanding-riboswitches/chen/978-0-12-801429-5"><font color="#330099">Elsevier Link</font></a>];[<a href="http://www.amazon.com/Computational-Understanding-Riboswitches-Developed-Structure/dp/0128014296"><font color="#330099">Amazon Link</font></a>] </p>
		    <p> Xu, XJ., Chen, SJ. (2015) A Method to Predict the 3D Structure of an RNA Scaffold. Methods Mol Biol. 1316: 1-11. doi: 10.1007/978-1-4939-2730-2_1.
<!-- In: Luc Ponchon (ed) 
RNA Scaffolds:  Methods and Protocols. 
Methods in Molecular Biology. -->
[<a href="paps/2015MiMB_scaffold_Chap1.pdf"><font color="#330099">PDF</font></a>]  
</p>
		    <p> Xu, XJ., Zhao, PN., Chen, SJ. (2014) Vfold: a web server for RNA structure and folding thermodynamics prediction. PLoS ONE doi: 10.1371/journal.pone.0107504. [<a href="paps/2014PLoS1VfoldServer.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Miao, Z., Blanchet, M. F., Boniecki, M., Bujnicki, J. M., Chen, SJ., Cheng, C., ... & Westhof, E. (2014). RNA-Puzzles Round II: assessment of RNA structure prediction of two large riboswitches. FEBS JOURNAL 281: 638-639. </p>
		    <p> Zhu, YH., Chen, SJ. (2014)  Many-body effect in ion binding to RNA. J. Chem. Phys. [<font color="#330099">Featured article</font>], 141, 055101 (2014); http://dx.doi.org/10.1063/1.489065 [<a href="paps/2014JCPmany_body.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Cao, S<sup>*</sup>, Xu, XJ,<sup>*</sup>, Chen, SJ. (2014) Predicting Structure and Stability for RNA Complexes with Intermolecular Loop-loop Base Pairing. RNA, 20:835-45. doi: 10.1261/rna.043976.113. <sup>*</sup>Equal first author. [<a href="paps/2014RNAKissingComplex.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> He, ZJ., Zhu, YH., Chen, SJ. (2014) Exploring Electrostatic Energy Landscape for Tetraloop-Receptor Docking. Phys. Chem. Chem. Phys., 16 (14), 6367-6375. http://xlink.rsc.org/?doi=C3CP53655F . [<a href="paps/2013PCCPTetraloopReceptor.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> K Tian, Z He, Y Wang, SJ Chen, LQ Gu (2013) Designing a Polycationic Probe for Simultaneous Enrichment and Detection of microRNAs in a Nanopore. ACS Nano, 2013, 7 (5), pp 3962–3969. DOI: 10.1021/nn305789z. http://pubs.acs.org/doi/abs/10.1021/nn305789z [<a href="paps/2013ACSNano.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> He, ZJ., Chen, SJ. (2013) Quantifying Coulombic and Solvent Polarization-Mediated Forces Between DNA Helices. J. Phys. Chem. B, 117 (24), pp 7221-7227. DOI: 10.1021/jp4010955.
http://pubs.acs.org/doi/abs/10.1021/jp4010955 [<a href="paps/2013JPCBHelixHydration.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Liu, L, Chen, SJ. (2012)  Coarse-grained prediction of RNA loop structures. PLoS ONE. http://dx.plos.org/10.1371/journal.pone.0048460[<a href="paps/2012PLoS1LoopStructure.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Xu, XJ, Chen, SJ. (2012) Kinetic mechanism of conformational switch between bistable RNA hairpins. Journal of the American Chemical Society, 134 (30), pp 12499-12507
DOI: 10.1021/ja3013819 . [<a href="paps/kin.12JACS.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Tan, ZJ, Chen, SJ. (2012) Ion-mediated RNA structural collapse: effect of spatial confinement. Biophysical Journal, 103 (4), 827-836.[<a href="paps/es.12BPJ.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> He, ZJ., Chen, SJ. (2012) Predicting ion-nucleic acid interactions by energy landscape-guided sampling.
Journal of Chemical Theory and Computation, 8 (6), pp 2095-2102. DOI: 10.1021/ct300227a [<a href="paps/es.12JCTC.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p>  Jose Almeida Cruz, Marc-Frederick Blanchet, Michal Boniecki, Janusz M. Bujnick, Shi-Jie Chen, Song Cao, Rhiju Das, Feng Ding, Nikolay V. Dokholyan, Samuel Coulbourn Flores, Lili Huang, Christopher A. Lavender, Veronique Lisi, Francois Major, Katarzyna Mikolajczak, Dinshaw J. Patel, Anna Philips, Tomasz Puton, John Santalucia, Fredrick Sijenyi, Thomas Hermann, Kristian Rother, Magdalena Rother, Alexander Serganov, Marcin Skorupski, Tomasz Soltysinski, Parin Sripakdeevong, Irina Tuszynska, Kevin M. Weeks, Christina Waldsich, Michael Wildauer, Neocles B. Leontis, and Eric Westhof (2012) RNA-Puzzles: A CASP-like evaluation of RNA three-dimensional structure prediction. RNA. 18 (4)  doi: 10.1261/rna.031054.111 
[<a href="paps/2012RNApuzzle.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Cao, S, Chen, SJ. (2012) Statistical mechanical modeling of RNA folding: from free energy landscape to tertiary structural prediction. Nucleic Acids Mol Biol. 27: 185-212. [<a href="http://link.springer.com/book/10.1007/978-3-642-25740-7//page/1"><font color="#330099">link</font></a>]</p>
		    <p> Cao, S., Chen, SJ. (2012) A domain-based model for predicting large and complex pseudoknotted structures. RNA Biology. 9: 201-212.  [<a href="paps/2012RNABIOLlargeRNA.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Cao, S., Chen, SJ. (2012) Predicting kissing interactions in microRNA–target complex and assessment of microRNA activity. Nucleic Acids Research doi: 10.1093/nar/gks052 [<a href="paps/2012NARmicroRNA.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p>  Zhao, PN., Zhang, WB., Chen, SJ. (2011) Cotranscriptional folding kinetics of RNA secondary structures. Journal of Chemical Physics, 135, 245101 (2011); http://dx.doi.org/10.1063/1.3671644 (10 pages)  [<a href="paps/kin.11JCP.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Chen, GS, Chen, SJ. (2011) Quantitative analysis of the ion-dependent folding stability of DNA triplexes. Physical Biology. 8: 066006 (9pp) [<a href="paps/es.11Pbio.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Cao, S., Chen, SJ. (2011) Structure and stability of RNA/RNA kissing complex: with application to HIV dimerization initiation signal. RNA. 17: 2130-2143. [<a href="paps/complex.11RNA.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Rockey, W.M., Hernandez, F.J., Huang, S.Y., Cao, S., Howell, C.A.,Thomas, G.S., Liu, X.Y., Lapteva, N., Spencer, D.M., McNamara II, J.O., Zou, X., Chen, SJ., Giangrande, P.H. (2011) Rational truncation of an RNA aptamer to prostate specific membrane antigen using computational structural modeling. Nucleic Acid Therapeutic, 21: 299-314. [<a href="paps/equil.11NAT.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Stammler, S.N., Cao, S., Chen, SJ, Giedroc, D.P. (2011) A conserved RNA pseudoknot in a putative molecular switch domain of the 3' untranslated region of coronaviruses is only marginally stable. RNA, 17: 1747-1759. [<a href="paps/equil.11RNAa.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Tan, ZJ, Chen, SJ. (2011) Salt contribution to RNA tertiary structure folding stability. Biophysical Journal, 101: 176-187. [<a href="paps/es.11BPJ.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Cao, S., Chen, SJ. (2011) Physics-based de novo prediction of RNA 3D structures.The Journal of Physical Chemistry B, 115:4216-26. [<a href="paps/equil.11JPC.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Cao, S., Fuertig, B., Schwalbe, Harald., Chen, SJ. (2010)
Folding Kinetics for the Conformational Switch between Alternative RNA Structures. The Journal of Physical Chemistry B, 114: 13609-13615. [<a href="paps/kin.10JPC.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2010) Predicting ion binding properties for RNA tertiary structures. Biophysical Journal, 99: 1565-1576 [<a href="paps/es.10BPJ.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p>  Liu, L., Chen, SJ. (2010) Computing the conformational entropy for RNA folds. Journal of Chemical Physics, 132, 235104; doi:10.1063/1.3447385 (12 pages) [<a href="paps/equil.10JCP.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p>  Zhao, PN., Zhang, WB., Chen, SJ. (2010) Predicting Secondary Structural Folding Kinetics for Nucleic Acids.  Biophysical Journal, 98: 1617-1625. [<a href="paps/kin.10BPJ.pdf"><font color="#330099">PDF</font></a>]  </p>
		    <p> Cao, S., Giedroc, DP., Chen, SJ. (2010) Predicting loop-helix tertiary structural contacts in RNA pseudoknots. RNA, 16: 538-552. [<a href="paps/equil.10RNA.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Chen, GS., Tan, ZJ., Chen, SJ. (2010) Salt dependent folding energy landscape of RNA three-way junction. Biophysical Journal, 98: 111-120. [<a href="paps/es.09BPJ.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ, Chen, SJ. (2010)  Importance of Diffuse Metal Ion Binding to RNA, Chapter 3 in "Structural and Catalytic Roles of Metal Ions in RNA" (volume of METAL IONS IN LIFE SCIENCES), edited by Astrid Sigel, Helmut Sigel, and Roland K. O. Sigel. Royal Society of Chemistry Publishing. [<a href="paps/review.11mils.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Cao, S., Chen, SJ. (2009) Predicting structures and stabilities for H-type pseudoknots with inter-helix loop. RNA, 15: 696-706. 
[<a href="paps/equil.09RNA.pdf"><font color="#330099">PDF</font></a>] 
[<a href="http://f1000.com/prime/1160999"><font color="#330099">Faculty of 1000 Recommended</font></a>]</p> 
		    <p> Cao, S., Chen, SJ. (2009) A new computational approach for mechanical folding kinetics of RNA hairpins. Biophysical Journal, 96: 4024-4034 [<a href="paps/kin.09BPJ.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ, Chen, SJ. (2009)  Predicting electrostatic forces in RNA folding, 465: 465-487, Chapter 22 in "Biophysical Approaches to RNA Structure and Folding" (volume of Methods in Enzymology), edited by Daniel Herschlag. [<a href="paps/es.09ME.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Chen, SJ. (2008) RNA Folding: Conformational Statistics, Folding Kinetics, and Ion Electrostatics. Annual Review of Biophysics 37: 197-214. (NIHMSID: 55566)[<a href="paps/review.08AnnuRev.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Cao, S., Chen, SJ. (2008) Predicting ribosomal frameshifting efficiency. Physical Biology 5, 016002: 1-10. (NIHMSID: 55567)[<a href="paps/equil.08Pbio.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2008) Electrostatic free energy landscapes for DNA helix bending. Biophysical Journal 94: 3137-3149. (NIHMSID: 55573)[<a href="paps/es.08BPJ1.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2008) Salt dependence of nucleic acid hairpin stability. Biophysical Journal 95: 738-752. (NIHMSID: 55574)[<a href="paps/es.08BPJ2.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Chen, SJ., Cao, S., Kopeikin, Z., Zhang, WB. Polymer Statistical Mechanics for RNA Folding. In "Concepts in Protein Computations", (Ed. Robert L. Jernigan, Andrzej Kloczkowski), Cambridge University Press. To be published. </p>
		    <p> Cao, S., Chen, SJ. (2007) Biphasic folding kinetics of RNA pseudoknots and telomerase RNA activity. Journal of Molecular Biology 367: 909-924. (PMCID: PMC1995092) [<a href="paps/kin.07JMB.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Tan, ZJ., Chen, SJ. (2007) RNA helix stability in mixed Na+/Mg2+ solution. Biophysical Journal 92: 3615-3632. (PMCID: PMC1853146) [<a href="paps/es.07BPJ.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2006) Electrostatic free energy landscape for nucleic acid helix assembly. Nucleic Acids Research 34: 6629-6639. [<a href="paps/es.06NAR.pdf"><font color="#330099">PDF</font></a>] [<a href="http://f1000.com/prime/1063759"><font color="#330099">Faculty of 1000 Recommended</font></a>]</p>
		    <p> Cao, S., Chen, SJ. (2006) Predicting RNA pseudoknot folding thermodynamics. Nucleic Acids Research 34: 2634-2652, 2006. (PMCID: PMC1463895) [<a href="paps/equil.06NAR.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2006) Ion-mediated nucleic acid helix-helix interactions. Biophysical Journal 91: 518-536. (PMCID: PMC1483110) [<a href="paps/es.06BPJ.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Kopeikin, Z., Chen, SJ. (2006) Folding thermodynamics of pseudoknotted chain conformations. Journal of Chemical Physics 124, 154903: 1-13. (NIHMSID: 55563) [<a href="paps/equil.06JCP.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p>Cao, S., Chen, SJ. (2006) Free energy landscapes of RNA/RNA complexes - with applications to snRNA complexes in spliceosomes. Journal of Molecular Biology 356: 292-312. (NIHMSID: 55564) [<a href="paps/complex.05JMB.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p>Zhang, WB., Chen, SJ. (2006) Exploring the complex folding kinetics of RNA hairpins. 2. effect of sequence, length, and misfolded states. Biophysical Journal 90: 778-787. (PMCID: PMC1367103) [<a href="paps/kin.06BPJ2.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p>Zhang, WB., Chen, SJ. (2006) Exploring the complex folding kinetics of RNA hairpins. 1. general folding kinetics analysis. Biophysical Journal 90: 765-777. (PMCID: PMC1367102) [<a href="paps/kin.06BPJ1.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2006) Nucleic acid helix stability: effects of salt concentration, cation valence and size, and chain length. Biophysical Journal 90: 1175-1190. (PMCID: PMC1367269) [<a href="paps/es.06BJ.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Chen, SJ., Tan, ZJ., Cao, S., Zhang, WB. (2006) Statistical mechanics of RNA folding. Physics (Chinese) 35: 218-229. (NIHMSID: 56190) [<a href="paps/review.05Phys.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Cao, S., Chen, SJ. (2005) Predicting RNA folding thermodynamics with a reduced chain representation model. RNA 11: 1884-1897. (PMCID: PMC1370876) [<a href="paps/equil.05RNA.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Kopeikin, Z., Chen, SJ. (2005) Statistical thermodynamics for chain molecules with simple RNA tertiary contacts. Journal of Chemical Physics 122, 094909: 1-13. (NIHMSID: 55565) [<a href="paps/equil.05JCP.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Tan, ZJ., Chen, SJ. (2005) Electrostatic correlations and fluctuations for ion binding to a finite length polyelectrolyte. Journal of Chemical Physics 122, 044903: 1-16. (NIHMSID: 55562) [<a href="paps/es.05JCP.pdf"><font color="#330099">PDF</font></a>] </p>
		    <p> Zhang, WB., Chen, SJ. (2003) Analyzing the folding rates and pathways using kinetic cluster method. Journal of Chemical Physics 119: 8716-8729. (NIHMSID: 55561) [<a href="paps/kin.03JCP2.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Zhang, WB., Chen, SJ. (2003) Master equation approach to finding the rate-limiting steps in biopolymer folding. Journal of Chemical Physics 118: 3413-3420. (NIHMSID: 55560) [<a href="paps/kin.03JCP1.pdf"><font color="#330099">PDF</font></a>]</p>
		    <p> Zhang, WB., Chen, SJ. (2002)  RNA hairpin-folding kinetics. Proceedings of the National Academy of Sciences U.S.A. 99: 1931-1936. [<a href="paps/kin.02PNAS.pdf"><font color="#330099">PDF</font></a>]</p>
               </section>
            </div>
         </div>
      <footer id="footer">
         <div class="container-fluid">
            <div class="row-fluid">
               <div class="span4">
                  <a href="https://physics.missouri.edu/"><img src = "images/MU_UnitSig_PhysicsandAstronomy_rgb_std_vert.png"/></a>
               </div>
              <div class="span4">
                  <a href="https://muidsi.missouri.edu/"><img src = "images/bioinfo_processed.png"/></a>
               </div>
               <div class="span4">
                  <a href="https://cafnr.missouri.edu/divisions/biochemistry/"><img src = "images/biochem.png"/></a>
               </div>
            </div>
         </div>
      </footer>
      <!-- Le javascript
         ================================================== -->
      <!-- Placed at the end of the document so the pages load faster -->
      <script src="js/jquery-1.9.1.min.js"></script>
      <script src="js/bootstrap.min.js"></script>
      <script>
         $(document).ready(function() {
             $(document.body).scrollspy({
                 target: "#navparent"
             });
         });
         
      </script>
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-12611547-1");
pageTracker._trackPageview();
} catch(err) {}</script>

<!-- Start of StatCounter Code -->
<script type="text/javascript">
var sc_project=5612792; 
var sc_invisible=1; 
var sc_partition=63; 
var sc_click_stat=1; 
var sc_security="a93c2e7b"; 
</script>

<script type="text/javascript"
src="http://www.statcounter.com/counter/counter.js"></script><noscript><div
class="statcounter"><a title="myspace visitor tracker"
href="http://www.statcounter.com/myspace/"
target="_blank"><img class="statcounter"
src="http://c.statcounter.com/5612792/0/a93c2e7b/1/"
alt="myspace visitor tracker" ></a></div></noscript>
<!-- End of StatCounter Code -->

   </body>
</html>
